markdown/ch3.markdown

```diff
@@ -620,7 +621,7 @@ Now we can try it out on the coins image:
 
 ```python
 sobel_mag = ndi.generic_filter(coins, sobel_magnitude_filter, size=3)
-plt.imshow(sobel_mag);
+plt.imshow(sobel_mag, cmap='viridis');
```

Comment:
Consistency with other plots. If you ask why it's not the default, our style file sets grayscale as the default.